artifacts/openshift/vdo-controller-manager-metrics-service-rbac.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
need to update Makefile as well for this change so the files are placed accordingly

Reply:
yes, will do these changes